### DISCUSSION TITLE
GitHub Copilot: Failed to initiate the GitHub login process. Please try again.

### DISCUSSION THREAD

**Discussion by modaye** [community/community#20361] [Copilot Conversations]
GitHub copilot cannot be logged in and used now. This has tried various methods on windows, but it never works. But after continuous attempts, it was finally successfully used on Ubuntu 20.04. What is the reason for this (utuntu20.04 is a virtual machine with shared network)
![image](https://user-images.githubusercontent.com/44232619/178397676-79cc7d67-5218-4a6a-96ed-4d3c892efd85.png)

**Comment by FahdSeddik** [comment]
Hey!
I think you might find an answer in this [link](https://github.com/github-community/community/discussions/18132). Hope it helps!

**Reply by modaye** [reply]
Hey, I've tried the answer to this link dozens of times. I tried it in different windows environments, but it didn't succeed. Always prompt 'retring GitHub token' and then the login is still unsuccessful. After restarting, the login is successful, but there is no code prompt, which is equivalent to no effect
